src/commands/circleci-continue-long-job-cancel.yml

```
@@ -11,6 +11,10 @@ parameters:
     description: the name of the environmental variable that contains the circleci token
     default: "CIRCLECI_TOKEN"
     type: string
+  ignore-jobs-containing:
+    description: do not warn people about workflows where all the on hold job names contain this string. Multiple strings seperated by comma, eg development,delete
```

```suggestion
    description: Do not warn about workflows where all of the on hold job names contain a string from this comma separated list
```

fwiw, I'd update all of the args to match, and the descriptions